iodata/test/test_wfn.py

```diff
@@ -78,7 +79,7 @@ def test_load_wfn_low_h2o():
     data = helper_load_wfn_low('h2o_sto3g.wfn')
     # unpack data
     title, atnums, atcoords, centers, type_assignments = data[:5]
-    exponents, mo_count, occ_num, mo_energy, coefficients, energy, _ = data[5:]
+    exponents, mo_count, occ_num, mo_energy, coefficients, energy, _, _ = data[5:]
```

Comment:
@msricher Would it make sense to add unit tests for the last two variables? They are now underscores but they correspond to `virial` and `mo_spin`, I believe.

Reply:
I don't know.  There was already one underscore when I started working on this.  And nobody wanted me to add more test files with the MOSPIN section.

Reply:
I'm not familiar with the MOSPIN section, so I can't comment on that. Would it still be useful to check the virial then?

Reply:
Yeah, it would be.  Sorry, my wrists are hurting today, I need to take a break.  So I can't do it for a few days...

Reply:
No problem and no rush either. Take all the time you need.

Reply:
OK, I added the tests for the virial.

Reply:
Thanks!